docs/conf.py

```diff
@@ -19,7 +19,7 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #sys.path.insert(0, os.path.abspath('.'))
-sys.path.insert(0, os.path.abspath('./_extensions'))
+sys.path.insert(0, os.path.abspath('./extensions_src'))
```

Comment:
Can this be s/extensions_src/extensions/ once again? I think just calling them extensions is good enough since we use them as-is and don't compile them into another asset unlike the diagrams for example.